gen/gen.go

```diff
@@ -34,6 +34,7 @@ package gen
 //go:generate go run github.com/golang/mock/mockgen -source=../internal/core/ports/room_storage.go -destination=../internal/adapters/room_storage/mock/mock.go -package=mock
 //go:generate go run github.com/golang/mock/mockgen -source=../internal/core/ports/instance_storage.go -destination=../internal/adapters/instance_storage/mock/mock.go -package=mock
 //go:generate go run github.com/golang/mock/mockgen -source=../internal/core/ports/operation_storage.go -destination=../internal/adapters/operation_storage/mock/mock.go -package=mock
+//go:generate go run github.com/golang/mock/mockgen -source=../internal/core/ports/operation_lease_storage.go -destination=../internal/adapters/operation_lease/mock/mock.go -package=mock
```

Comment:
@lyracampos @caiordjesus Every time that we create a new interface that we want to mock, we have to add its target here so the `make generate` command can generate the mock file automatically.

Reply:
Awesome! Thanks for the information!